src/display/displayTemplateScale.h

```diff
@@ -76,11 +74,11 @@ void printScreen() {
             u8g2.print(totalBrewTime / 1000, 1);
         }
 
-        #if (FEATURE_PRESSURESENSOR == 1)
-            u8g2.setCursor(32, 46);
-            u8g2.print("P: ");
-            u8g2.print(inputPressure, 1);
-        #endif
+#if (FEATURE_PRESSURESENSOR == 1)
+        u8g2.setCursor(32, 46);
+        u8g2.print("P: ");
+        u8g2.print(inputPressure, 1);
+#endif
```

Comment:
this lost the intendation after void printScreen. But that ist the problem with the makros you said?

Reply:
Exactly. If we change the `#if` macros for proper `if(){}` statements this will be fixed. See above.